### DISCUSSION TITLE
I'm being kicked out of my lobby in multiplayer osu lazer

### DISCUSSION THREAD

**Discussion by WakariLain** [ppy/osu#30844] [Q&A]
I make a room in multiplayer and sometimes I leave in the middle of the game, it happened several times and this appeared:
[1732130266.runtime.log](https://github.com/user-attachments/files/17874452/1732130266.runtime.log)

**Comment by bdach** [comment]
You're getting kicked out because your connection is unstable. Nothing we can do about that.
